src/App.tsx

```diff
@@ -70,7 +70,7 @@ export function App() {
 						/>
 						<Route
 							path="/manage-list"
-							element={<ManageList listPath={listPath} />}
+							element={<ManageList listPath={listPath} data={data || []} />}
 						/>
```

Comment:
suggestion: I don't think wee need to pass `data || []` in the data prop, I beleive the data is initialized as an empty array when creating a list. So a list would either have a filled array or an empty array.

Reply:
Ahhh do you mean we wouldn't need to pass the prop at all or not need to the OR operator to suggest it could be empty?

Reply:
Sorry that was very unclear 😅 my bad I think you can have it just as `data={data}` instead of `data={data || []}`

Reply:
Noted!! That makes sense